### PR TITLE
feat(bench): Add unified GEMM performance testing

### LINKED ISSUE #16: Performance: Check the tiled CUDA performance after modifying the swizzle.

### PR DESCRIPTION
**Pull request by KuangjuX**
Resolved #16 .

### CODE REVIEW

File: benchs/gemm/gemm_bench_NVIDIA_A100_80GB_PCIe.csv [hunk 1]
```diff
@@ -0,0 +1,14 @@
+M,N,K,kTM,kTN,kTK,cuBLAS(ms),Cutlass(ms),TiledCUDA(ms)
+4096,4096,2048,128,256,64,0.27764737606048584,1.56727294921875,nan
```
Comment (haruhi55) [inline benchs/gemm/gemm_bench_NVIDIA_A100_80GB_PCIe.csv]: Format the output to truncate to the given precision, such as 4 decimal places. Long numbers are inconvenient to read.

File: benchs/gemm/bench.py [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+    K: int,
+    time: Tensor
+):
+    # Run unittest
+    if run_cublas_unittest(a, b, c, M, N, K):
+        # print("Unittest passed")
+        pass
+    else:
+        # raise ValueError("Unittest failed")
+        print("Run cuBLAS unittest failed")
```
Comment (haruhi55) [inline benchs/gemm/bench.py:121]: Do you still want to keep lines 120 and 123?

File: benchs/gemm/bench.py [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+    kTN: int,
+    kTK: int,
+    warp_layout: Tuple,
+):
+    if run_cutlass_unittest(a, b, c, M, N, K, kTM, kTN, kTK, warp_layout):
+        # print("Unittest passed")
+        pass
+    else:
+        # raise ValueError("Unittest failed")
+        print("Run Cutlass unittest failed")
```
Comment (haruhi55) [inline benchs/gemm/bench.py:146]: Do you still want to keep lines 147 and 150?

File: benchs/gemm/bench.py [hunk 4]
```diff
@@ -0,0 +1,274 @@
...
+    kTN: int,
+    kTK: int,
+    kRK: int,
+    warp_layout: Tuple,
+):
+    if run_tiledcuda_unittest(a, b, c, M, N, K, kTM, kTN, kTK, kRK, warp_layout):
+        pass
+    else:
+        # raise ValueError("Unittest failed")
+        print("Run TiledCUDA unittest failed")
```
Comment (haruhi55) [inline benchs/gemm/bench.py:186]: Do you still want to keep lines 191?

File: benchs/gemm/gemm_bench_NVIDIA_A100_80GB_PCIe.csv (hunk 1 above)
Comment (KuangjuX) [reply benchs/gemm/gemm_bench_NVIDIA_A100_80GB_PCIe.csv]: fixed.

File: benchs/gemm/bench.py (hunk 2 above)
Comment (KuangjuX) [reply benchs/gemm/bench.py:121]: fixed.

File: benchs/gemm/bench.py (hunk 3 above)
Comment (KuangjuX) [reply benchs/gemm/bench.py:146]: fixed.

File: benchs/gemm/bench.py (hunk 4 above)
Comment (KuangjuX) [reply benchs/gemm/bench.py:186]: fixed.

**Review by haruhi55 — APPROVED** [review]
LGTM

**Pull request merged**